### DISCUSSION TITLE
Chives not showing up correctly.

### DISCUSSION THREAD

**Discussion by Yamanipanuchi** [guydavis/machinaris#369] [Q&A] [answered]
I know its related to my setup, But for some reason Chives does not show up correctly when rotating through the different coins.

![image](https://user-images.githubusercontent.com/4724631/138909410-4bdf21dd-b04d-4fe9-b65c-e81165bbeed6.png)

The top bar does not show up that shows chives. I am sure it is purely cosmetic, but I am wonder what I can do to fix it?

**Comment by guydavis** [comment, marked as answer]
Hi, so the screenshot you show indicates the Machinaris-Chives fullnode is running and connected to the Machinaris-Chia fullnode/controller.  A screenshot of your Workers page would make this clear.

However, Machinaris is showing you as having zero Chives plots.  If you are wanting to plot for Chives, please see the [wiki](https://github.com/guydavis/machinaris/wiki/Chives).  

If instead, you already have  some Chives plots, you need to be sure your volume mounts of the host path match to your in-container paths.  

You don't say in your original question, so I'm going to guess you are running Linux.  If so, please share the complete docker-compose.yml you generated at https://www.machinaris.app.

**Reply by Yamanipanuchi** [reply]
Yes, I haven't copied any chive plots over to this system yet, Will be by the end of the day. Mainly noticed at the top it doesn't show chives for this tab. Chia, Flax, Nchain and HDDcoin do show at the top.

Running an unraid system, So I didn't use the Docker-Compose feature.

**Comment by Yamanipanuchi** [comment]
Okey, Once I actually got Chives on the machine is all looks normal.
